src/utils/body.ts

```diff
@@ -195,10 +240,15 @@ export async function readMultipartFormData(event: H3Event) {
  *  }
  * ```
  */
-export async function readFormData(event: H3Event) {
+export async function readFormData(event: H3Event): Promise<FormData> {
   return await toWebRequest(event).formData();
 }
 
+/**
+ * Captures a stream from a request.
```

Comment:
It'd be great to have a better definition what exactly a "web request" is for people not aware that there are various web APIs ☺️

Reply:
Surely! Will keep in mind for when having official docs to explain